### PR TITLE
Add Meal as a value for key to form implementation

### PR DESCRIPTION
**Pull request by WarenaReveal**
I found my condition wasn't working due to not explicitly setting key as the value of Meal

## What is the purpose of the change

My conditional XOR gate was not working in the tutorial. I found that it was due to me not adding Meal as the key value. I dont think the tutorial included this.

## Are there related marketing activities

No

## When should this change go live?

Entirely up to you

## PR Checklist

- [Yes ] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [Yes ] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [No] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [Yes] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by akeller** [comment]
Thanks for your PR @WarenaReveal! Can you please sign our CLA?

I've added @Camundaan and @christinaausley to your PR for reviews.

File: versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md [hunk 1]
```diff
@@ -54,8 +54,9 @@ Variables are part of a process instance and represent the data of the instance.
 1. Navigate to the Modeler **Home** page and click the project you are working on. Then, click **New > Form**.
 2. Name your form. In this case, we've named ours **Decide what's for dinner**.
 3. Click and drag the **Select** element onto the palette. Give this **Select** field a description within the properties panel. We've described ours as **What's for dinner?**
-4. Scroll down to the **Values** section of the properties panel to add your values. For our dinner add an extra value by clicking on the plus sign. Enter the value **Chicken** with the label as **Chicken** and enter the value **Salad** with the label as **Salad** in the other value.
-5. Within your BPMN diagram, click the appropriate user task, and click the blue square icon in the bottom right corner of your user task. Select the form you'd like to apply to your user task, and click **Import** to add the form to the BPMN diagram.
+4. Add **Meal** as the key value
```
Comment (christinaausley) [inline versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md:57]: How about:
"4. Under the **General** tab of the properties panel, enter **Meal** as the **Key**."

Is this what you mean?

**Review by christinaausley — CHANGES_REQUESTED** [review]
One comment for clarification -- thanks for opening this PR!

File: versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md (hunk 1 above)
Comment (WarenaReveal) [reply versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md:57]: Hi @christinaausley. Yep under the general tab. I should have been more specific.

GeneralTab

Comment (WarenaReveal) [reply versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md:57]: This one is Dinner. For another example I did. But wanted to make sure I was explaining the right area.

Comment (christinaausley) [reply versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md:57]: Perfect! If you could adjust to clarify, I'll go ahead and approve. I'm headed off on vacation, but @Camundaan can also approve once the adjustment is made.

Comment (WarenaReveal) [reply versioned_docs/version-8.0/guides/getting-started-orchestrate-human-tasks.md]: Awesome, done and done. Enjoy your holiday!

Cheers

**Review by christinaausley — APPROVED** [review]

**Comment by akeller** [comment]
Thanks for your PR @WarenaReveal! I'll get this merged.

**Pull request merged**